CHANGELOG.md

```diff
@@ -22,9 +25,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [1.10.0-0.29b0](https://github.com/open-telemetry/opentelemetry-python/releases/tag/v1.10.0-0.29b0) - 2022-03-10
 
-- `opentelemetry-instrumentation-wsgi` Capture custom request/response headers in span attributes
```

Comment:
Did this not make the previous release?

Reply:
yes but the position of this statement was wrong so I just corrected it